docs/ipac/irsa/irsa.rst

```diff
@@ -220,6 +220,62 @@ returned as a `~pyvo.dal.TAPResults` object. Its ``to_table`` or ``to_qtable`` m
     202.809023  46.964558  15.874     0.081 ...  15.322     0.188     AAC    000
 
 
+Simple image access queries
+---------------------------
+
+`~astroquery.ipac.irsa.IrsaClass.query_sia` provides a way to access IRSA's Simple
+Image Access VO service. In the following example we are looking for Spitzer
+Enhanced Imaging products in the centre of the COSMOS field as an `~astropy.table.Table`.
+
+.. doctest-remote-data::
+
+   >>> from astroquery.ipac.irsa import Irsa
+   >>> from astropy.coordinates import SkyCoord
+   >>> from astropy import units as u
+   >>>
+   >>> coord = SkyCoord('150.01d 2.2d', frame='icrs')
+   >>> spitzer_images = Irsa.query_sia(pos=(coord, 1 * u.arcmin), collection='spitzer_seip').to_table()
+
+Now open a cutout image for one of the science images. You could either use
+the the IRSA on-premise data or the cloud version of it using the
+``access_url`` or ``cloud_access`` columns:
+
+.. doctest-remote-data::
+
+   >>> from astropy.io import fits
+   >>> from astropy.nddata import Cutout2D
+   >>> from astropy.wcs import WCS
+   >>> science_image = spitzer_images[spitzer_images['dataproduct_subtype'] == 'science'][0]
+   >>> with fits.open(science_image['access_url'], use_fsspec=True) as hdul:
```

Comment:
Could you add a line above explaining when/whether `use_fsspec` is required?  i.e., is it required for both on-prem & cloud versions?

Reply:
this is all a copy from the astropy docs, so I might just link there. (The cloud version is a bit more complicated as ``cloud_access`` is a json string and its parsing is not yet added into pyvo. 
At irsa have docs about how to use it with pyvo, so I don't think it's necessary to repeat it here just yet.

(also, `query_sia` follows what is done in `alma`, but it's not at all user friendly, both the parameter names and their behaviors closely follow pyvo's API, so ultimately I think I need to write a wrapper for `get_images` or similar. But I plan to have that once 0.4.7 is out rather than hold up it any longer).